microcosm_eventsource/models/rollup.py

```diff
@@ -12,10 +12,21 @@ class RollUp:
     object and that specific event attributes will be exposed via @property syntax.
 
     """
-    def __init__(self, event, container, rank, *args, **kwargs):
+    def __init__(self, event, container, aggregate):
```

Comment:
Any aggregate defined by a rollup now gets saved in as an underscored attribute in this model.